modules/deploy/partials/high-availability.adoc

```diff
@@ -371,6 +371,149 @@ ID HOST PORT RACK
+* Complexity: The RedPanda operational complexity is not appreciably increased, but the complexity of your overall cloud solution is increased. Maintaining presence across availability zones requires additional servers with corresponding maintenance, access control, and other considerations.
+
+=== Multi-AZ infrastructure
+
+When we talk about multiple availability zones, RedPanda requires a minimum of three zones. Deploying across only two availability zones is problematic. For example, given a cluster with three brokers spread across two availability zones, you either end up with all three brokers in one zone or a pair of brokers in one with a single broker in the other. Either way, it's possible to lose a majority of your brokers with a single availability zone outage. You lose the ability to form consensus in affected partitions, negating the high availability state you desire.
+
+=== Multi-AZ optimization
+
+You can configure xref:develop:consume-data/follower-fetching.adoc[follower fetching] to help ease the latency problems associated with a multi-AZ configuration. This is achieved by configuring consumers to advertise their preferred rack using the `client.rack` option within their consumer configuration. This allows consumers to read data from their closest replica rather than always reading from a (potentially non-local) partition leader. Using this capability has the additional benefit of reducing non-local network traffic and thereby minimizing any cross-AZ network charges.
+
+=== Multi-AZ example
+
+Creating a multi-AZ deployment in RedPanda cloud is very straightforward - just select the Multi AZ availability option when creating a cluster. RedPanda also provides an official https://github.com/redpanda-data/deployment-automation[deployment automation] project using Ansible and Teraform to help self-hosted users stand up multi-AZ deployments quickly and efficiently.
+
+==== Configuring terraform
+
+The first thing yu need to do is to configure the appropriate terraform script for your cloud provider. Within the deployment-automation project, locate the file for your cloud of choice and edit the availability_zones parameter. Include each availability zone you intend to use for your deployment. For example, under AWS, you will edit the `aws/main.tf` file:
+
+```bash
+variable "availability_zone" {
+description = "The AWS AZ to deploy the infrastructure on"
+default     = ["us-west-2a", "us-west-2b", "us-west-2c"]
+type        = list(string)
+}
+```
+
+Alternatively, you can supply the configuration at the command line as follows:
+
+```bash
+$ terraform apply -var=availability_zone='["us-west-2a","us-west-2b","us-west-2c"]'
+```
+
+==== Deploying with terraform and ansible
+
+Once you have teraform configured for your cloud provider and chosen availability zones, you can proceed to deploying your cluster. This example steps through deploying a multi-AZ cluster and validating the rack configuration.
```

Comment:
```suggestion
After you configure Terraform for your cloud provider and choose availability zones, you can deploy your cluster. The following example deploys a multi-AZ cluster and validates the rack configuration.
```

Reply:
@greg-szrama Please implement the feedback from @micheleRP . I'm seeing that her comments are marked resolved, but the changes have not been implemented. I can circle back to review after--please ping me then. Thx